### PR TITLE
[mlu]add mlu place docs

### PR DESCRIPTION
**Pull request by fwenguang**
add docs of MLUPlace，is_compiled_with_mlu。

https://github.com/PaddlePaddle/Paddle/pull/38292

### CODE REVIEW

**Comment by qili93** [comment]
文档预览链接见： http://10.136.157.23:8090/documentation/docs/zh/faq/index_cn.html?reviewVersion=jenkins-doc-review-1873

中文文档见：

image

image

image

image

image

英文文档见：

image

image

image

image

image

File: docs/api/paddle/static/mlu_places_cn.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.. py:function:: paddle.static.mlu_places(device_ids=None)
+
+
+.. note::
+    多卡任务请先使用 FLAGS_selected_mlus 环境变量设置可见的MLU设备。
+
+该接口根据 ``device_ids`` 创建一个或多个 ``paddle.MLUPlace`` 对象，并返回所创建的对象列表。
+
+如果 ``device_ids`` 为 ``None``，则首先检查 ``FLAGS_selected_mlus`` 标志。
+例如： ``FLAGS_selected_mlus=0,1,2`` ，则返回的列表将为 ``[paddle.MLUPlace(0), paddle.MLUPlace(1), paddle.MLUPlace(2)]``。
```
Comment (TCChenlong) [inline docs/api/paddle/static/mlu_places_cn.rst]: paddle.MLUPlace -> paddle.device.MLUPlace

Comment (fwenguang) [reply docs/api/paddle/static/mlu_places_cn.rst]: DONE

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**